instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import CommentInput from "./CommentInput";
+import Comment from "./Comment";
+import moment from "moment";
+import styled from 'styled-components';
+
+const CommentsSection = styled.section`
+    display: flex;
+    flex-direction: column;
+`;
+
+const TimeStamp = styled.span`
+    color: silver;
+    font-size: .8rem;
+    padding-left: 15px;
+`;
+
+const BorderTop = styled.hr`
+    color: rgb(199, 198, 198);
+    width: 96%;
+    margin: 15px auto; 
+`;
+
+class CommentSection extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: props.comments,
+            comment: "", 
+
+        }
+    }
+
+    addNewComment = e => {
+        e.preventDefault(); 
+        if(!this.state.comment) return;
+        let newComment = { 
+            username: localStorage.getItem('username'), 
+            text: this.state.comment
+        };
+        this.setState({
+                comments: [...this.state.comments, newComment],
```

Comment:
Here we would use a method passed down from `PostPage` to update our data there and remove line 42